extensions/smallrye-reactive-messaging-pulsar/deployment/src/test/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+mp.messaging.outgoing.source.connector=smallrye-pulsar
+
+mp.messaging.incoming.in.connector=smallrye-pulsar
```

Comment:
unrelated to that PR - I'm started to think we should drop `smallrye-`. It can be confusing. (both value should work)